src/MeetingsSDKAdapter.js

```diff
@@ -238,13 +237,20 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
    * @returns {Observable.<MeetingControlDisplay>}
    * @memberof MeetingJSONAdapter
    */
-  muteAudioControl(ID) {
+  audioControl(ID) {
     const sdkMeeting = this.fetchMeeting(ID);
     const muted = {
-      ID: MUTE_AUDIO_CONTROL,
+      ID: AUDIO_CONTROL,
+      icon: 'microphone-muted',
+      tooltip: 'Unmute',
+      state: MeetingControlState.ACTIVE,
+      text: null,
+    };
+    const unmuted = {
+      ID: AUDIO_CONTROL,
       icon: 'microphone',
```

Comment:
I just re-checked UX and for unmute state the icon should also be `microphone-muted`